Interfaces/API/Dirac.py

```diff
@@ -75,6 +81,39 @@ def __init__( self, withRepo = False, repoLocation = '', useCertificates = False
 
     self.__clients = None
 
+  def __checkFileArgument( self, fnList, prefix = None, single = False ):
```

Comment:
And we fought to get rid of the 'single' argument in the DM, do you really want to reintroduce it here ?